test_fractions.py

```diff
@@ -796,11 +797,50 @@ def test_pi_digits(self):
             self.assertEqual(ff.numerator, qf.numerator)
             self.assertEqual(ff.denominator, qf.denominator)
 
+class CImportTest(unittest.TestCase):
+
+    def setUp(self):
+        self.build_test_module()
```

Comment:
This is redundant with the `test_cimport()` method. I think the test methods are the right place to trigger the module build, because they would probably want to build different modules.

Reply:
Ah, I see. I duplicated `self.build_test_module()` in [test_cimport](https://github.com/scoder/quicktions/blob/efcca1567f8cdefeae5ef4e4bf92721071511717/test_fractions.py#L827-L828).  Again, I'm always a little out of sorts with unittest-style

Reply:
Feel free to write `pytest` tests in the `test_quicktions.py` module. As I mentioned, the only reason why `test_fractions.py` looks the way it does is that it's copied from CPython's test suite.